guides/login-with-github.mdx

```diff
@@ -0,0 +1,116 @@
+## Login With GitHub
+
+If you've read the [Login With Google guide](/guides/login-with-google), you would have learned that Flipt supports many methods of authentication for users to control who has access to what in Flipt.
```

Comment:
Slight wording adjustment to avoid implying we support any kind of authz yet.

```suggestion
If you've read the [Login With Google guide](/guides/login-with-google), you would have learned that Flipt supports many methods of authentication for users to control who has access to Flipt.
```